spec/views/conferences/show.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe "conferences/show", type: :view do
+  before(:each) do
+    @conference = assign(:conference, Conference.create!(
+      :name => "Name"
```

Comment:
Indent the first parameter one step more than `Conference.create!(`.<br>Use the new Ruby 1.9 hash syntax.